spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -26,25 +26,71 @@
     let(:policy_machine_storage_adapter) { described_class.new }
 
     describe 'find_all_of_type' do
+      it 'accepts an array parameter on a column attribute' do
+        search_uuids = [SecureRandom.uuid, SecureRandom.uuid]
+        policy_machine_storage_adapter.add_object(search_uuids[0], 'some_policy_machine_uuid1')
+        policy_machine_storage_adapter.add_object(search_uuids[1], 'some_policy_machine_uuid1')
+        policy_machine_storage_adapter.add_object(SecureRandom.uuid, 'some_policy_machine_uuid1')
+
+        expect(
+          policy_machine_storage_adapter.find_all_of_type_object(
+            unique_identifier: search_uuids
+          ).count
+        ).to eq(2)
+      end
 
-      it 'warns once when filtering on an extra attribute' do
-        expect(Warn).to receive(:warn).once
-        2.times do
-          expect(policy_machine_storage_adapter.find_all_of_type_user(foo: 'bar')).to be_empty
+      context 'when case insensitive' do
+        it 'accepts an array parameter on a column attribute' do
+          pm_uuid = SecureRandom.uuid
```

Comment:
I would just use a let block up after the describe since this is referenced a lot`let(:pm_uuid) { SecureRandom.uuid }`